examples/jupyter/demo002_diffusion.ipynb

```diff
@@ -97,7 +97,7 @@
     "        ylabel=\"spectral radius\",\n",
     "        linewidth=3,\n",
     "        legend=:none,\n",
-    "        title=\"Mass Operator Symbol\"\n",
+    "        title=\"Diffusion Operator Symbol\"\n",
```

Comment:
Some of these changes are already in main. I think this branch may need to be rebased